appveyor.yml

```diff
@@ -1,52 +0,0 @@
-
```

Comment:
I need to go through the jobs on AppVeyor and ensure they all have the skip is no ```appveyor.yml``` config set.